package.json

```diff
@@ -21,10 +21,12 @@
     "@nestjs/typeorm": "^2.0.0",
```

Comment:
do we have yarn added to this project? I forget. If we do will you add these with yarn so it updates the yarn.lock. If not don't worry about it I'll add it later.